fix: Load `SalesChannelCategoryEntity` when loading the breadcrumb to properly access the seo links (#9254)
shopware/platform: changelog/_unreleased/2025-05-10-load-saleschannelcategoryentity-when-loading-the-breadcrumb.md

---
title: Load `SalesChannelCategoryEntity` when loading the breadcrumb
author: Max
author_github: @aragon999
---
# Core
* Changed the Twig functions `sw_breadcrumb_full()` and `sw_breadcrumb_full_by_id()` to load `SalesChannelCategoryEntity` in order to properly display the corresponding seo link in the breadcrumb
___
# Upgrade Information

> change in the Twig templates

Reviewer: this needs a `##` headline, so it could be merged with the other changelogs. 

everything under `# Upgrade Information` will be merged together, so it would look strange, that this flies around somewhere

Author: @mitelg mh, I added the information to the `UPGRADE-6.8.md` file. Is this then not required? Then I would replace the information from there in this changelog.

Reviewer: this changelog file should contain how to upgrade now, to prevent the deprecation. the 6.8 upgrade guide should document what will be removed or changed and how to fix that, if not already done

Reviewer: yes, it could be kind of duplicated, but people will not remember how to fix that with 6.8, just they read it here on some 6.7 release :wink:

Author: Okay, not sure, but is it is now in the PR correct? Or did I misunderstood anything? :)

Reviewer: is okay now :+1:

Author: Perfect :-)

> The Twig breadcrumb functions `sw_breadcrumb_full` and `sw_breadcrumb_full_by_id` now require the `SalesChannelContext`, i.e. adjust the default Twig templates as follows

Reviewer: ```suggestion
The Twig breadcrumb functions `sw_breadcrumb_full` and `sw_breadcrumb_full_by_id` will require the `SalesChannelContext` with the next major version.
Adjust the Twig templates as follows
```
